packages/agoric-cli/src/commands/vaults.js

```diff
@@ -59,6 +77,116 @@ export const makeVaultsCommand = async logger => {
       }
     });
 
+  vaults
+    .command('info')
```

Comment:
I would have expected `governance` or `params` but I see you're outputting metrics too. I can't think of a good reason someone would want only one and not the other so this lgtm. (the sometimes superfluous network request is nbd)